js/modules/loki_snode_api.js

```diff
@@ -6,6 +6,9 @@ const dns = require('dns');
 const process = require('process');
 const { rpc } = require('./loki_rpc');
 const natUpnp = require('nat-upnp');
+const LokiMixpanelAPI = require('./loki_mixpanel.js');
+
+const Mixpanel = new LokiMixpanelAPI();
```

Comment:
Just use `window.mixpanel` directly in the code below

Reply:
addressed